docs/zh/integration/offline_data_sources/s3.md

```diff
@@ -0,0 +1,27 @@
+# AWS S3
+
+## 简介
+
+[AWS S3](https://docs.aws.amazon.com/s3/index.html) 是 AWS 云上的对象存储服务。 OpenMLDB 支持使用 S3 作为离线存储引擎，用于读取和导出特征计算的数据。
+
+## 配置
+
+### 配置Taskmanager
+
+要读写 AWS S3 的数据，首先需要在 AWS 云服务上注册账号，申请一个可以用的 AccessKey 和 SecretKey。
+
+然后在 TaskManager 的配置文件 `taskmanager.properties` 中添加下面的配置即可，注意需要修改 AccessKey 和 SecretKey 为上面申请的密钥。
+
+```
+spark.default.conf=spark.hadoop.fs.s3a.access.key=xxx;spark.hadoop.fs.s3a.secret.key=xxx
+```
+
+## 使用
+
+### 导入数据
+
+如果想要导入 S3 的数据，在上面 TaskManager 配置成功并且重启后，可直接执行下面的命令，即可导入对应 Bucket 和路径的数据。
```

Comment:
do we support all the options of LOAD DATA INFILE for S3? such as deep_copy (I guess we only support `true`?), mode? Please clarify those available options.

Reply:
ok